docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  bot:
+    restart: unless-stopped
+    container_name: bot
+    image: containers.pkg.github.com/Webeleon/discord-bot-starter
+    environment:
+      PORT: 5000
```

I'd rather use the .env file instead of passing configurations to the compose file.
```suggestion
      PORT
```

the other solution would be to use the `env_file` directive and point to the .env file
[compose documentation on env variables](https://docs.docker.com/compose/environment-variables/)

---

This is resolved in newest push